.github/workflows/pkgdown.yaml

```diff
@@ -36,18 +38,20 @@ jobs:
         run: |
           pak::local_install(upgrade = TRUE)
           pak::local_install_dev_deps()
+          pak::pkg_install("r-lib/pkgdown")
+          pak::pkg_install("apreshill/quillt")
         shell: Rscript {0}
 
       - name: Build pkgdown site
-        run: |
-          pak::pkg_install("pkgdown")
-          pkgdown::build_site(new_process = FALSE)
+        run: pkgdown::build_site(new_process = FALSE)
         shell: Rscript {0}
 
-      - name: Publish
-        run: |
-          npm install netlify-cli -g
-          netlify deploy --prod --dir docs
+      - name: Deploy to Netlify
+        if: false
+        # NETLIFY_AUTH_TOKEN added in the repo's secrets
         env:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
+        run: |
+          npm install netlify-cli -g
+          netlify deploy --prod --dir docs
```

Comment:
I deactivated this part because I do not want the PR to overwrite the current website. 

@apreshill I need your skill with netlify (and because I don't think I have access to RStudio Netlify org) so that we have a preview from this PR (and not just from your fork)

Can we have a preview deploy ?